src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+.page {
+  --active-star: url(/src/images/star-active.svg);
+}
+
+.page__body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :first-child {
```

Comment:
The selector `.stars--1 :first-child` has a space before `:first-child`, which means it targets the first child of any element inside `.stars--1`. If the intention is to target the first `.stars__star` directly inside `.stars--1`, remove the space: `.stars--1:first-child`.